### PR TITLE
Make `config` command support abbreviation

### LINKED ISSUE #532: config command does not accept abbreviation
**Description**

**Steps to reproduce the issue**
1. 
```
admin@sonic:~$ sudo config interf startup Ethernet2
```

**Describe the results you received**
```
Usage: config [OPTIONS] COMMAND [ARGS]...

Error: No such command "interf".
```

**Describe the results you expected**
It should support command abbreviation.

**Additional information you deem important (e.g. issue happens only occasionally)**

**Output of `show version`**
```
SONiC Software Version: SONiC.20181130.20
Distribution: Debian 9.9
Kernel: 4.9.0-8-amd64
Build commit: ea1a21d
Build date: Tue May  7 07:59:02 UTC 2019
Built by: sonicbld@jenkins-slave-phx-3

Docker images:
REPOSITORY                 TAG                 IMAGE ID            SIZE
docker-syncd-brcm          20181130.20         3a92a514fc96        361MB
docker-syncd-brcm          latest              3a92a514fc96        361MB
docker-orchagent-brcm      20181130.20         4a4e0b959307        286MB
docker-orchagent-brcm      latest              4a4e0b959307        286MB
docker-lldp-sv2            20181130.20         01c23b57577c        274MB
docker-lldp-sv2            latest              01c23b57577c        274MB
docker-dhcp-relay          20181130.20         c2fcb935dc7e        256MB
docker-dhcp-relay          latest              c2fcb935dc7e        256MB
docker-database            20181130.20         af6a03ecc883        255MB
docker-database            latest              af6a03ecc883        255MB
docker-teamd               20181130.20         384afb01b8fa        274MB
docker-teamd               latest              384afb01b8fa        274MB
docker-snmp-sv2            20181130.20         d9b6f79bb453        289MB
docker-snmp-sv2            latest              d9b6f79bb453        289MB
docker-router-advertiser   20181130.20         161a2bf893a8        254MB
docker-router-advertiser   latest              161a2bf893a8        254MB
docker-platform-monitor    20181130.20         c65fd90fc91e        286MB
docker-platform-monitor    latest              c65fd90fc91e        286MB
docker-fpm-quagga          20181130.20         4e31139f8d11        281MB
docker-fpm-quagga          latest              4e31139f8d11        281MB
```

### PR DESCRIPTION
**Pull request by qiluo-msft**
Fix #532

### CODE REVIEW

File: config/main.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from swsssdk import ConfigDBConnector
 from swsssdk import SonicV2Connector
 from minigraph import parse_device_desc_xml
+from click_default_group import DefaultGroup
```
Comment (jleveque) [inline config/main.py:18]: Is it necessary to import click_default_group to accomplish this? We don't use the functionality of DefaultGroup any longer, and I was planning to remove it entirely. Can instead inherit a Click class?

Comment (jleveque) [reply config/main.py:18]: DefaultGroup inherits from `click.Group`. Can you try simply inheriting this class and see if this works?

Comment (jleveque) [reply config/main.py:18]: We can merge this as-is and attempt to replace DefaultGroup with click.Group in a separate PR.

**Comment by abdosi** [comment]
@jleveque Please create PR for 201911. There are conflicts.

**Comment by jleveque** [comment]
@abdosi: 201911-specific PR here: https://github.com/Azure/sonic-utilities/pull/933

**Pull request merged**